config/locales/cy.yml

```diff
@@ -225,87 +225,6 @@ toplevel:
           rhaglen benodol ar ein gwefan ddydd Llun.
         </p>
       linkText: Darllenwch ein diweddariadau COVID-19 diwethaf
-  wales:
```

Comment:
I can't believe I forgot to delete this when I moved these pages to the CMS. You just stole these 200 lines of deletion from me.

Reply:
sorry not sorry